modules/ROOT/pages/creating-document-actions.adoc

```diff
@@ -78,7 +78,7 @@ Add Prompts to refine the results of the extraction by asking questions about th
 ** *Instruction*: a question or request in natural language.
 +
 [WARNING]
-Prompts don't support special characters.
+Prompts don't support special characters, ensure you use alphanumeric characters when writing your prompts. This limitation doesn't apply to documents, which can contain any type of characters or symbols. 
```

Comment:
Should this info be added in Quotas and Limits page too? Additionally, can we also cover that prompt supports English only?